### DISCUSSION TITLE
Is vaultwarden incompatible with bw-cli?

### DISCUSSION THREAD

**Discussion by 766F6964** [dani-garcia/vaultwarden#2083] [Q&A]
I am currently running vaultwarden with the recommended docker-compose setup. I tried to interact with my vaultwarden instance using the bitwarden cli tool.

I specified my server in bw-cli, and was able to successfully log in.

However, for every bw-cli command (for example list items, list folders etc), an empty list is returned, although there are items in the vault. Why does that happen?

Am I missing something or does bw-cli not work with vaultwarden?

**Comment by jjlin** [comment]
It should work fine, and does for me. It would be helpful if you specified the exact commands you used.
